pkgs/development/node-packages/generate.sh

```diff
@@ -3,7 +3,7 @@ set -eu -o pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 node2nix=$(nix-build ../../.. -A nodePackages.node2nix)
-cd ${DIR}
+cd "${DIR}"
```

Comment:
```suggestion
cd "$DIR"
```

Reply:
Would you like me to squash the commits?